build/versioninfo.ps1

```diff
@@ -7,5 +7,5 @@ param(
 [string]$version
 )
 $versionComma = $version.Replace(".", ",")
-$files = 'Texassemble\texassemble.rc', 'Texconv\Texconv.rc', 'Texdiag\texdiag.rc'
+$files = 'Texassemble\texassemble.rc', 'Texconv\Texconv.rc', 'Texdiag\texdiag.rc', 'build\DirectXTex.rc.in'
```

Comment:
This is for completeness should I ever decided to add building the DLL version as part of the NuGet pipeline some day.